release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -538,20 +540,43 @@ git push origin merged-master-release-[current release]
 
 Now create a pull request to merge the new branch you've made into the `release-[current-release]` branch on [k/website](https://github.com/kubernetes/website).
 
-### Freeze Kubernetes website
+#### Update the `config.toml`s for the past four releases
```

Comment:
nit: Update the `config.toml` files for the ... OR
Update the site configuration files